c2corg_ui/templates/i18n.html

```diff
@@ -376,5 +376,8 @@
 <span translate>has updated the article</span>
 <span translate>has updated the route</span>
 <span translate>has updated the image</span>
+<span translate>has added photos to the route</span>
+<span translate>has added photos to the outing</span>
+<span translate>has added photos to the waypoint</span>
```

Comment:
no because it will not work (or we don't know how to) for example with german translations : 

`<span translate>has added photos to the</span><span translate>outing</span>`
...hat Fotos zum Tourenbericht hinzugefügt.